package.json

```diff
@@ -2,6 +2,13 @@
   "name": "chess-game",
   "version": "0.1.0",
   "private": true,
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "lint": "eslint ./src/**/*.js"
+  },
   "dependencies": {
     "@testing-library/jest-dom": "^4.2.4",
```

Comment:
Will these testing ones be moved to devDependencies with Avery's PR? I saw eslint dependencies as well - do those need to be moved to devDependencies as well?

Reply:
It looks like these testing packages are placed in "dependencies" rather than "devDependencies" by default when a react app is created. Seems safe to move them so they don't get bundled